package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pw-tech/platform-hyper-express",
-  "version": "0.1.2",
+  "version": "0.1.3",
```

Comment:
should be a minor version, as its a feature

Reply:
done